lib/registry/src/config.rs

```diff
@@ -1,26 +1,25 @@
 use graphql_client::GraphQLQuery;
-use serde::Deserialize;
-use serde::Serialize;
+use serde::{Deserialize, Serialize};
 use std::collections::BTreeMap;
 use std::path::{Path, PathBuf};
 
 #[derive(Deserialize, Default, Serialize, Debug, PartialEq, Eq)]
-pub struct PartialWapmConfig {
+pub struct WasmerConfig {
     /// The number of seconds to wait before checking the registry for a new
     /// version of the package.
     #[serde(default = "wax_default_cooldown")]
     pub wax_cooldown: i32,
 
-    /// The registry that wapm will connect to.
-    pub registry: Registries,
-
     /// Whether or not telemetry is enabled.
     #[serde(default)]
-    pub telemetry: Telemetry,
+    pub telemetry_enabled: bool,
 
     /// Whether or not updated notifications are enabled.
     #[serde(default)]
-    pub update_notifications: UpdateNotifications,
+    pub update_notifications_enabled: bool,
+
+    /// The registry that wapm will connect to.
+    pub registry: Registries,
```

Comment:
I have no idea what registries is here

Reply:
What we should do here is:

Have a field tokens that associate a given registry with a token.

We should also have a new field called: current registry.


Config should be like

```json
tokens = [{
    registry: "https://registry.wapm.dev/graphql",
    token: "abc"
}, ...]
active_registry = "https://registry.wapm.io/graphql"
```